src/site/layout/header.rs

```diff
@@ -70,6 +71,15 @@ fn nav(
         html.extend(html!(<li><a href=href>{text!("Docs")}</a></li>));
     };
 
+    if *changelog {
```

Comment:
is this `*` required?

Reply:
When not using it I get this error

![image](https://user-images.githubusercontent.com/1051509/222754738-ecd7a3e2-4ea1-42bb-9c9e-bd1b96be0f53.png)